docs/coordinates/spectralcoord.rst

```diff
@@ -0,0 +1,530 @@
+is still as before, but the observer velocity is now ~10-20 km/s in x, y, and z,
+which is because the observer is now stationary relative to the barycenter so has
+a significant velocity relative to the surface of the Earth.
+
+We can also transform the frequencies to the LSRD frame of reference::
+
+    >>> sc_ttau.with_observer_velocity('lsrd')  # doctest: +REMOTE_DATA +FLOAT_CMP
+    <SpectralCoord [200.01820327, 210.01911343, 220.02002359, 230.02093376,
+                    240.02184392, 250.02275408, 260.02366425, 270.02457441,
+                    280.02548457, 290.02639474, 300.0273049 ] GHz
+        observer:
+          <LSRD Coordinate: (x, y, z) in m
+              (-1.25867767e+11, -7.48979688e+10, -3.24757657e+10)
+           (v_x, v_y, v_z) in km / s
+              (0., 0., 0.)>
+        target:
+          <ICRS Coordinate: (ra, dec, distance) in (deg, deg, pc)
+              (65.497625, 19.53511111, 144.321)
+           (pm_ra_cosdec, pm_dec, radial_velocity) in (mas / yr, mas / yr, km / s)
+              (1.37949782e-15, 1.46375638e-15, 23.9)>
+        observer to target (computed from above):
+          radial_velocity=13.751182544045982 km / s
+          redshift=4.587005966327773e-05>
+
+See :ref:`spectralcoord-common-frames` for a list of common velocity frames
+available as strings on the |SpectralCoord| class.
+
+Since we can give any arbitrary |SkyCoord| to the
+:meth:`~astropy.coordinates.SpectralCoord.with_observer_velocity`
+method, we can also specify the target itself, to find the frequencies in the
+rest frame of the target::
+
+    >>> sc_ttau_targetframe = sc_ttau.with_observer_velocity(sc_ttau.target)  # doctest: +REMOTE_DATA
+    >>> sc_ttau_targetframe  # doctest: +REMOTE_DATA +FLOAT_CMP
+    <SpectralCoord [200.02737999, 210.02874899, 220.03011799, 230.03148698,
+                    240.03285598, 250.03422498, 260.03559398, 270.03696298,
+                    280.03833198, 290.03970098, 300.04106998] GHz
+        observer:
+          <ICRS Coordinate: (x, y, z) in m
+              (-1.25867767e+11, -7.48979688e+10, -3.24757657e+10)
+           (v_x, v_y, v_z) in km / s
+              (9.34149908, 20.49579745, 7.99178839)>
+        target:
+          <ICRS Coordinate: (ra, dec, distance) in (deg, deg, pc)
+              (65.497625, 19.53511111, 144.321)
+           (pm_ra_cosdec, pm_dec, radial_velocity) in (mas / yr, mas / yr, km / s)
+              (1.37949782e-15, 1.46375638e-15, 23.9)>
+        observer to target (computed from above):
+          radial_velocity=0.0 km / s
+          redshift=0.0>
+
+
+The ``radial_velocity``, which is the velocity offset between observer and
+target, is now zero.
+
+|SpectralCoord| is very versatile and can be used to represent any spectral
+values - not just the x-axis of a spectrum, but also for example the
+frequencies of spectral features. For example, if we now consider that we found a
+spectral feature that appears to have three features at the following frequencies
```

Comment:
double "feature" - maybe "spectral feature that has components at the following three ..."

Reply:
Done